config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-release-1.25.yaml

```diff
@@ -74,7 +99,7 @@ presubmits:
         - runner.sh
         env:
         - name: KUBERNETES_VERSION
-          value: "v1.25.0"
+          value: "v1.25.12"
```

Comment:
Not yet available at https://hub.docker.com/r/kindest/node/tags?page=1&name=1.25